lua/auto-save/config.lua

```diff
@@ -9,7 +9,11 @@ Config = {
       dim = 0.18, -- dim the color of `message`
       cleaning_interval = 1250, -- (milliseconds) automatically clean MsgArea after displaying `message`. See :h MsgArea
     },
-    trigger_events = { "InsertLeave", "TextChanged" }, -- vim events that trigger auto-save. See :h events
+    trigger_events = { -- See :h events
+      immediate_save = { "BufLeave", "FocusLost" }, -- vim events that trigger an immediate save
```

Comment:
Not sure about "FocusLost" here but then again don't see any hard reason not to save immediately then so maybe that's just me.

Reply:
What's your reason against `FocusLost`? I have some obscure workflows (editing with `nvim` in tab 1, then switching to tab 2 and running the tests from there). This was the reason for including this :)

Ultimately we should maybe add a section to the README, that includes information for useful events.